src/main/resources/config/en.yml

```diff
@@ -286,11 +286,15 @@ DiscordCannedResponses: {"!ip": "yourserveripchange.me", "!site": "http://yoursi
 #
 # MinecraftDiscordAccountLinkedRoleNameToAddUserTo: the name or id of a discord role to add a discord user to when they link their account
 # MinecraftDiscordAccountLinkedAllowRelinkBySendingANewCode: allows sending a new code to the bot to unlink and relink with the new code
+# MinecraftDiscordAccountEnablePM: allows linking accounts using PM
+# MinecraftDiscordAccountChannels: the name or id of a discord channel to link accounts
 #
 MinecraftDiscordAccountLinkedConsoleCommands: ["", "", ""]
 MinecraftDiscordAccountUnlinkedConsoleCommands: ["", "", ""]
 MinecraftDiscordAccountLinkedRoleNameToAddUserTo: "Linked"
 MinecraftDiscordAccountLinkedAllowRelinkBySendingANewCode: false
+MinecraftDiscordAccountEnablePM: true
+MinecraftDiscordAccountChannels: []
```

Comment:
We already have a `Channels` option, I feel it would be fine to just allow linking in the already defined channels

Reply:
I would like to have a dedicated channel for linking on my server and guide people to link on the channel for linking.

So I will try to implement an additional option to link when writing to Channels as well.

Reply:
I'd advise adding a "link account in any registered chat channel" option and looking up a `link` named channel in the `Channels` config for a dedicated link channel otherwise

Reply:
I understand. I'll try.

Reply:
Okay, thanks

Reply:
If so, how would you specify if you wanted to limit it to PMs only?

Reply:
`MinecraftDiscordAccountEnablePM: true` is fine to me as-is

Reply:
Ok,
`MinecraftDiscordAccountEnablePM: true` → Only PMs are allowed to link.
`MinecraftDiscordAccountEnablePM: false` → Only text channels are allowed to link.

Reply:
Channel linking should be available at the same time PM linking is, the PM linking option should just control whether people can/can't link with a PM

Reply:
You are right.
I have found that I cannot mix a channel for chat and a link channel.
This is because the feature of replying with an invalid link code warning interferes with the ability to chat.
Therefore, the link channel and the other channels need to be separated.

So I'll implement the following
`MinecraftDiscordAccountEnablePM: true/false` → enable/disable PMs to link.
`Channels`.`link` → empty for disable, fill channel id for enable text channels to link.

Reply:
I fixed it.